charts/redhat/redhat/knative-istio-authz/OWNERS

```diff
@@ -0,0 +1,13 @@
+chart:
+  name: knative-istio-authz
+  shortDescription: This is the Red Had Openshift Serverless knative-istio-authz chart
```

Comment:
Helm chart to onboard a set of authorization-based isolated namespaces to Knative when using Istio